rp_gpconnect/tasks.py

```diff
@@ -95,3 +95,23 @@ def import_or_update_contact(patient_info, org_id):
             restart_participants=True,
             extra=patient_info,
         )
+
+
+@app.task(
+    autoretry_for=(HTTPError, ConnectionError, Timeout, SoftTimeLimitExceeded),
```

Comment:
I don't think that we're making any HTTP calls in here, so we can probably remove the HTTP errors from this list.

Also, since this task happens every hour, I'm not sure that we need retries for this. Rather have it fail and try again in an hour, than have the tasks queue up because they're all stuck retrying when something is broken.